### ISSUE TITLE
Keras fit_generator using validation does not respect verbose argument

### ISSUE THREAD

**Issue by geometrikal** [tensorflow/tensorflow#28648]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 1809
- TensorFlow installed from (source or binary): pip
- TensorFlow version (use command below): 1.13 gpu
- Python version: 3.6

**Bug**
When using `fit_generator` with validation the progress bar is printed regardless of the `verbose` setting.

**Cause**
Line 216 of tensorflow/python/keras/engine/training_generator.py calls `model_iterator` to do validation, but does not pass the `verbose` parameter, meaning that the default value of `verbose=1` is used.

**Solution**
Add in the missing parameter, e.g.

```
    # Run the test loop every epoch during training.
    if do_validation and not callbacks.model.stop_training:
      val_results = model_iteration(
          model,
          validation_data,
          steps_per_epoch=validation_steps,
          batch_size=batch_size,
          class_weight=class_weight,
          workers=workers,
          use_multiprocessing=use_multiprocessing,
          max_queue_size=max_queue_size,
          verbose=verbose,
          mode='test')
```

**Comment by jvishnuvardhan** [comment]
@geometrikal Could you provide a standalone code to reproduce the issue? Thanks!

**Comment by geometrikal** [comment]
From #28738 
```
import tensorflow as tf
import numpy as np

model = tf.keras.models.Sequential([
    tf.keras.layers.Dense(1, input_shape=(1,))
])
model.compile(loss="mae", optimizer="adam")

def generator():
    i=0
    while 1:
        yield (np.array([i]),[i])
        i+=1
valData = (np.arange(10), np.arange(10))

history = model.fit_generator(generator(), steps_per_epoch=5, verbose=0, validation_data=valData)
```

**Comment by omalleyt12** [comment]
Hi @geometrikal, thanks for the issue! As you have a solution, this would be a good opportunity to submit a PR if you'd like. Would you like to?

Otherwise I'll look into the fix

**Comment by geometrikal** [comment]
Closing because this appears fixed in 1.14
